backend/src/main/java/com/ody/notification/domain/Notification.java

```diff
@@ -63,4 +63,8 @@ public static Notification createNudge(Mate mate) {
     public void updateStatusToDone() {
         this.status = NotificationStatus.DONE;
     }
+
+    public boolean isStatusDismissed() {
```

Comment:
[의견]

enum type은 public static이니 의미만을 담고 메서드로 한번더 물어보지 않아도 된다고 생각해요! 위의 updateStatusToDone에도 이미 Notification이 NotificationStatus를 알고 있다고 생각했구요!

다음과 같은 구조는 어떤가요?

```suggestion
    public boolean isStatusDismissed() {
            return status == NotificationStatus.DISMISSED
    }
```

Reply:
콜리말도 동의하지만 Enum에서 boolean 메서드를 정의해서 재사용하는 경우도 많은 것 같아서 지금도 좋은 것 같아요 !

Reply:
[반영] Enum에서 boolean 메서드를 정의해서 사용하는 경우가 있어서 위와 같이 코드를 작성했는데 사실 제가 좋아하는 방식은 아니라 콜리의 리뷰를 반영했습니다 😁 [cbff59f](https://github.com/woowacourse-teams/2024-ody/pull/512/commits/cbff59fd4493041d6def017cf82686e4f944bbcd)